BETA_TESTING.md

```diff
@@ -1,47 +1,66 @@
+This page describes the fastest way to build DevTools with the goal to use it. Do not mix this setup with development environment. If you want to make code changes, follow [contributing guidance](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md).
 
-If you want to make code changes, follow [contributing guidance](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md).
+You may want to build DevTools to:
 
-The steps were tested for Mac. They may require adjustments for other platforms.
+1. Try experimental features
+
+2. Run desktop version, instead of Web version, to get get rid of browser memory limit. For example,
+to be able to analyze heap snapshots of more complicated applications.
+
+The steps were tested for Mac and may require adjustments for other platforms. Contributions,
+that make the steps more platform agnostic, are welcome.
 
 ## Prerequisites
 
 1. [Configure](https://docs.flutter.dev/get-started/install) Dart or Flutter.
 
-## Setup and start
+## Setup DevTools
+
+If it is initial setup:
 
 1. In your terminal `cd` to a folder where you want to clone devtools, and that does not have subfolder `devtools` yet.
 
-2. Clone the repo, get needed Flutter version to local folder and start DevTools:
+2. Clone the repo and get needed Flutter version to local folder:
 
 ```bash
 git clone https://github.com/flutter/devtools.git
 
-./devtools/tool/update_flutter_sdk.sh
-
-cd devtools/packages/devtools_app
-../../tool/flutter-sdk/bin/flutter run -d chrome --dart-define=enable_experiments=true
+bash devtools/tool/update_flutter_sdk.sh
+cd devtools
 ```
 
-3. Paste the URL of your application (for example [Gallery](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md#connect-to-application)) to the connection textbox.
+## Or refresh DevTools
 
-## Refresh and start
+If you are refreting previously configured version:
 
 1. `cd` to the `devtools` directory created in the [Setup and start](#setup-and-start) section.
 
-2. Refresh and run DevTools (it will delete all your local changes!):
+2. Refresh DevTools (it will delete all your local changes!):
 
 ```bash
 git checkout master
 git reset --hard origin/master
 
-./tool/update_flutter_sdk.sh
-cd packages/devtools_app
-../../tool/flutter-sdk/bin/flutter pub upgrade
-
-../../tool/flutter-sdk/bin/flutter run -d chrome --dart-define=enable_experiments=true
+bash tool/update_flutter_sdk.sh
+bash tool/upgrade.sh
 ```
 
-3. Paste the URL of your application (for example [Gallery](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md#connect-to-application)) to the connection textbox.
+If some steps failed, remove the directory and redo to [Setup](#setup).
+
+## Start DevTools
+
+1. `cd packages/devtools_app`
+
+2. Start DevTools
+
+In Chrome: `../../tool/flutter-sdk/bin/flutter run --release -d chrome`
+On Mac: `../../tool/flutter-sdk/bin/flutter run --release -d macos`
+On Windows: `../../tool/flutter-sdk/bin/flutter run --release -d windows`
```

Comment:
can these be sub-bullets?
prefix each line with ' - '

also r/In Chrome/On Chrome for consistency